security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/PostContingencyResultSerializer.java

```diff
@@ -27,9 +27,9 @@ public void serialize(PostContingencyResult postContingencyResult, JsonGenerator
         jsonGenerator.writeStartObject();
         jsonGenerator.writeObjectField("contingency", postContingencyResult.getContingency());
         jsonGenerator.writeObjectField("limitViolationsResult", postContingencyResult.getLimitViolationsResult());
-        jsonGenerator.writeObjectField("branchResults", postContingencyResult.getBranchResults());
-        jsonGenerator.writeObjectField("busResults", postContingencyResult.getBusResults());
-        jsonGenerator.writeObjectField("threeWindingsTransformerResults", postContingencyResult.getThreeWindingsTransformerResult());
+        jsonGenerator.writeObjectField("branchResults", postContingencyResult.getNetworkResult().getBranchResults());
```

Comment:
I think we should change the JSON serialization so that it's the same as in the new objects.

It means that, in the deserializer, we need to handle version 1.1 and version 1.2 files differently on those fields (in 1.2 they will be in networkResult).